tests-rtl/src/store.js

```diff
@@ -0,0 +1,21 @@
+import { initialState as schemaInitialState } from "./api";
+import { LogContext, each, log, parallel, take } from "starfx";
+import { configureStore } from "starfx/store";
+
+export function setupStore({ initialState = {} }) {
+  const store = configureStore({
+    initialState: {
+      ...schemaInitialState,
+      ...initialState,
+    },
+  });
+
+  const tsks = [];
+
+  store.run(function* () {
+    const group = yield* parallel(tsks);
+    yield* group;
+  });
```

Comment:
```suggestion
  store.run(api.bootup);
```

Reply:
Ahh! Missed that in my copy and pasting. I wonder if we could somehow through a dev only error there, it hasn't been the first time and likely not the last that that is missed.

Reply:
Hmm yes I think you are on the right track.  We want the user to "do the right thing" and building some error handling like "no tasks were activated" could be useful.